<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Browser</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>411</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDownload">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>490</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTest01">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>381</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>Test 01</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxSaveMeta">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>490</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Metadata</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAddLayer">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>510</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Add as a layer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
